packages/addons/addon-depends/jre-depends/jdk-x86_64-zulu/package.mk

```diff
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2019-present Peter Vicman (peter.vicman@gmail.com)
+
+PKG_NAME="jdk-x86_64-zulu"
+PKG_VERSION="8u212"
```

Comment:
PKG_VERSION not used at URL ? I see there is some meh versioning going on but maybe use at least `8.38.0.13` ?

Reply:
8u212 is OpenJDK version which this zulu binary is based on. But will change to zulu binary version like 8.38.0.13 in this case.